rpm/native_db.go

```diff
@@ -412,22 +411,16 @@ func constructHint(b *strings.Builder, info *Info) string {
 				if p.SigType != 0 {
 					continue
 				}
-				if b.Len() != 0 {
-					b.WriteByte('|')
```

Comment:
instead of appending, we will now overwrite the `key` if we get both `*packet.SignatureV3` and `*packet.Signature`. Is this possible, and if so, is overwriting expected?

Reply:
That should never happen because a given `rpm` version should only be producing one version of signature. If they ever do get mixed, I'm fine with last-one-wins.